.asf.yaml

```diff
@@ -54,7 +54,7 @@ github:
     - SreeramGarlapati
     - samredai
```

Comment:
Actually, if all is the same to @samredai, would you mind swapping his name instead? I still plan on doing some docs work but didn't want to hold down the list, but I'm pretty sure Sam is on to other things.

If this is too much of a bother then happy to merge this now and I'll add a new PR later to swap with Sam's name when I need it.

Reply:
Oh, I might have misunderstood your words. Then let me cancel this PR.